messaging.rst

```diff
@@ -175,39 +58,58 @@ Invariants
-+======================+===============+============================================================+
-|  balance_proof       | BalanceProof  | Balance proof for this transfer                            |
-+----------------------+---------------+------------------------------------------------------------+
++----------------------+---------------+----------------------------------------------------------------+
+| Field Name           | Field Type    |  Description                                                   |
++======================+===============+================================================================+
+|  nonce               | integer       | Sequential nonce per participant.                              |
++----------------------+---------------+----------------------------------------------------------------+
+|  token               | address       | Address of the token being exchanged.                          |
++----------------------+---------------+----------------------------------------------------------------+
+|  transferred_amount  | integer       | Total amount transferred to the partner since channel opened.  |
++----------------------+---------------+----------------------------------------------------------------+
+|  initiator           | address       | Raiden node initiating the transfer.                           |
++----------------------+---------------+----------------------------------------------------------------+
+|  recipient           | address       | Raiden node receiving the transfer.                            |
++----------------------+---------------+----------------------------------------------------------------+
+|  locksroot           | bytes         | Root of a Merkle tree of currently outstanding locks.          |
++----------------------+---------------+----------------------------------------------------------------+
+
+The ``nonce`` is counted up by each participant to order messages and further protect against replay attacks.
 
 Mediated Transfer
 -----------------
 
+.. Note:: At the current (11/07/2018) raiden implementation as of commit ``04eeecce9ee875bdbed96fe7db85c2a1ece57803``, the ``LockedTransfer`` class is used for this message.
+
 Cancellable and expirable :term:`transfer`. Sent by a node when a transfer is being initiated, this message adds a new lock to the corresponding merkle tree of the sending participant node.
 
 
 Invariants
 ^^^^^^^^^^
 
-- The :term:`balance proof` locksroot must be equal to the previous valid merkle tree with the lock provided in the messaged added into it.
-- The transfer is valid only if the lock amount is smaller than the sender's :term:`capacity`.
+- The :term:`locksroot` must be equal to the previous valid merkle tree with the lock provided in the messaged added into it.
+- The transfer is valid only if the ``locked_amount`` is smaller than the sender's :term:`capacity`.
```

Comment:
Same for other occurrences - I would use the balance proof concept and keep the initial version.